container/os-autoinst_dev/Dockerfile

```diff
@@ -129,6 +128,7 @@ RUN zypper in -y -C \
        'perl(Test::Pod)' \
        'perl(Test::Strict)' \
        'perl(Test::Warnings)' \
+       'perl(Text::Levenshtein)' \
```

Comment:
Unfortunately the SR has been declined. Maybe @perlpunk can help with that.

Reply:
Oops, I thought that SR would be created differently. Looks like I could not handle it myself then.
It would be great if @perlpunk could help with it.

Reply:
https://build.opensuse.org/request/show/1112337

Reply:
Thanks a lot!